CHANGELOG.md

```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [Preview Release 5.2.0-preview5.24024.5] - 2024-01-24
```

Comment:
the value 5.2.0-preview5.24024.5 may change. Just keep an eye before merging the pipeline.